projects/pre-commit.com/package.yml

```diff
@@ -0,0 +1,47 @@
+distributable:
+  url: https://github.com/pre-commit/pre-commit/archive/refs/tags/v{{ version }}.tar.gz
+  strip-components: 1
+
+# if there’s a github then we can parse the versions
```

Comment:
we should remove the generated comments.

Reply:
done